lua/terrortown/entities/items/item_ttt_radar/cl_init.lua

```diff
@@ -271,32 +251,42 @@ end
 net.Receive("TTT_CorpseCall", TTT_CorpseCall)
 
 local function ReceiveRadarScan()
-	local num_targets = net.ReadUInt(8)
+	local num_targets = net.ReadUInt(16)
 
 	RADAR.targets = {}
 
 	for i = 1, num_targets do
-		local sr = net.ReadUInt(ROLE_BITS)
+		local pos = Vector(
+			net.ReadInt(32),
```

Comment:
Shouldn't `net.Write/ReadFloat` make more sense?

Reply:
the positions are rounded. I don't know why, but it has been like this forever.

Reply:
Yea I know that you didn't change it, but it doesn't make sense to me. And if they are rounded, a 32 bit int? o.O

Reply:
yes, they are indeed rounded. Isn't a float 32bit as well? Doesn't make sense to me.

Reply:
No I mean I can't imagine that a position of "2147483647" is available on a map :D That would break anything if it would be possible (because 2147483648 isn't far away). I like the solution with rounding somehow, but 32 bits yea...

Reply:
![image](https://user-images.githubusercontent.com/13639408/81182887-b1e6dd00-8fae-11ea-87a7-2dc1f5362da7.png)

source: https://www.reddit.com/r/gmod/comments/74eu8c/how_limited_is_the_map_size/